docs/content/_index.md

```diff
@@ -3,38 +3,34 @@ title: Brigade Documentation
 description: Homepage of the Brigade Documentation
 ---
 
-# Brigade Documentation
-
-Everything you need to know about Brigade.  
-
-### First Steps
+## First Steps
```

Comment:
We skipped right over the h2's here and went to h3's for some reason, so this PR fixes that also.